ansible/site.yml

```diff
@@ -18,6 +18,19 @@
   roles:
     - logging
     - gateway
+  tasks:
+    - shell: >
+        ip address show dev ens4
+        | grep -oP 'inet (\d+(\.\d+){3})'
+        | awk '{print $2}'
+      register: GCP_NETWORK_INTERFACE_GATEWAY_IP_ADDRESS
+    - shell: >
+        ip address show dev wg0
+        | grep -oP 'inet (\d+(\.\d+){3})'
+        | awk '{print $2}'
+      register: WIREGUARD_NETWORK_INTERFACE_GATEWAY_IP_ADDRESS
+  vars:
+    K8S_CLUSTER_API_IP_ADDRESS: 172.10.0.2
```

Comment:
This var is already provided as `tunnel_ip` (as used here: https://github.com/nais/device/blob/master/ansible/roles/gateway/templates/gateway-agent.service.j2#L18)

Reply:
I missclicked here, i mean the WireGuard IP.
Keep the K8S ip.

Reply:
> This var is already provided as `tunnel_ip` (as used here: https://github.com/nais/device/blob/master/ansible/roles/gateway/templates/gateway-agent.service.j2#L18)

Yeah, I saw that it was in use but I could not find its definition.
I could not decipher the purpose of the variable from the name `tunnel_ip` alone - at least not so that I felt confident as to what it contained.

I will accept this review, but then make an issue to clarify variable(s) to include purpose in name, as well as description of data.